d2renderers/d2svg/d2svg.go

```diff
@@ -1435,6 +1440,11 @@ func drawShape(writer, appendixWriter io.Writer, diagramHash string, targetShape
 		el.Fill = fill
 		el.Stroke = stroke
 		el.Style = style
+		if targetShape.IconBorderRadius != 0 {
+			clipPathId := fmt.Sprintf("%v-%v-icon", diagramHash, svg.SVGID(targetShape.ID))
```

Comment:
convention in this codebase is to double uppercase ID, `clipPathID` please (everywhere in PR)